packages/mdc-floating-label/_mixins.scss

```diff
@@ -68,6 +68,38 @@
   }
 }
 
+@mixin mdc-floating-label-outlined-float-position($positionY, $positionX: 0%, $scale: .75) {
+  .mdc-floating-label--float-above {
+    font-size: ($scale * 1rem);
+
+    @if $positionX > 0 or $positionX < 0 {
+      transform: translateY(-1 * $positionY - 14%) translateX(-1 * $positionX) scale(1);
```

Comment:
What's the significance of 14%? Is there some existing variable this is related to?

Reply:
Since I'm changing the font-size it changes the amount of translateY needed to get the label centered vertically into the notch. This is simply a static amount to adjust for the 25% reduction in font-size. I'll move it into a variable when I move this into the notched-outline package.